src/u2f/u2f_app.c

```diff
@@ -139,6 +161,16 @@ static void _app_string(const uint8_t* app_id, char* out, size_t out_len)
             return;
         }
     }
+    char* mnemonic=NULL;
+    //fallback to hex if something goes wrong
+    if (bip39_mnemonic_from_bytes(NULL, app_id, 32, &mnemonic) == WALLY_OK) {
+        const char* words[SHORT_MNEMONIC_LENGTH];
+        //second fallback
+        if(short_mnemonic(mnemonic, words)>=4) {
```

Comment:
use the define here instead of `4`

Reply:
okay, I tried to generalize where possible to make the function also possible to be used at other places if the need arises while also making sure that someone changing the constant to fit other needs while forgetting this in here (which is due to the sprintf kinda hardwired to a min of 4) to make sure these 4 are in to prevent it crashing